### ISSUE TITLE
Debugger entered--Lisp error: (void-function evil-leader/set-key-for-mode)

### ISSUE THREAD

**Issue by Enrico68** [edwtjo/evil-org-mode#23]
When I try to `eval-buffer` `evil-org.el` I get this error:

````
Debugger entered--Lisp error: (void-function evil-leader/set-key-for-mode)
  (evil-leader/set-key-for-mode (quote org-mode) "t" (quote org-show-todo-tree) "a" (quote org-agenda) "c" (quote org-archive-subtree) "l" (quote evil-org-open-links) "o" (quote evil-org-recompute-clocks))
  eval-buffer()  ; Reading at buffer position 5095
  call-interactively(eval-buffer record nil)
  command-execute(eval-buffer record)
  execute-extended-command(nil "eval-buffer")
  smex-read-and-run(("toggle-debug-on-error" "eval-buffer" "auto-complete-mode" "apri-url-con-macosx-browser" "calendar" "helm-themes" "helm-w3m-bookmarks" "eshell" "customize-group" "mml-attach-file" "password-store-copy" "toggle-frame-fullscreen" "make-frame" "customize-variable" "ebib" "notmuch-unread-mode" "bug-hunter-init-file" "flycheck-mode" "password-store-edit" "graphviz-dot-preview" "password-store-generate" "quickrun" "elpy-mode" "ebib-import" "notmuch-hello" "maximize-frame" "bbdb" "delete-frame" "org-mobile-push" "iedit-mode" "artist-mode" "python-mode" "auto-complete" "server-start" "auto-fill-mode" "kivy-mode" "run-python" "helm-colors" "recover-file" "maximize-window" "pdf-tools-install" "dklrt-AppendRecurring" "sunshine-toggle-icons" "esup" "malyon" "sx-answer" "bbdb-snarf" "helm-bibtex" "flyspell-mode" "restore-frame" ...))
  smex()
  call-interactively(smex nil nil)
  command-execute(smex)
`````

**Comment by patterkyle** [comment]
I had this same problem. Do you have evil-leader (https://github.com/cofi/evil-leader) installed? It's listed as one of the requirements for evil-org-mode. Installing it fixed the issue for me.

**Comment by TheBB** [comment]
evli-org-mode requires evil-leader, but it's not listed as a package dependency. This is a bug.

**Comment by edwtjo** [comment]
Agreed, fixed in 61319f85979e8768c930983595caa2483c0fb319
